src/site/_includes/global/prismjs-theme.scss

```diff
@@ -0,0 +1,129 @@
+@use 'sass:color';
```

Comment:
Custom theme for prism.js with GOV.UK colours – had to darken some for contrast (against the grey background).